CHANGELOG.md

```diff
@@ -5,8 +5,25 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [0.9.0] - 2023-08-14
+### Fixed
+- Fix lua-check warnings
+- Fix peer dependencies warning
+- Replace `@types/commander` with `commaner`
+- Remove and update npm packages for security reason
+
+### Added
+- Added `dep` and `depclean` target
```

Comment:
`dep` and `depclean` are no longer supported, since we don't install `npm` modules anymore.

Reply:
`commander` is no longer required.